src/number/types/Fixed18.sol

```diff
@@ -21,7 +34,7 @@ library Fixed18Lib {
     /// @custom:error Arithmetic overflow
     error Fixed18OverflowError(uint256 value);
 
-    int256 private constant BASE = 1e18;
+    int256 internal constant BASE = 1e18;
     Fixed18 public constant ZERO = Fixed18.wrap(0);
```

Comment:
should we make these ones `internal` as well? I didn't really constants only need to be internal to be used outside of the library.